src/features/operator/Market/MarketWithdraw/index.tsx

```diff
@@ -201,6 +376,10 @@ export const MarketWithdraw = (): JSX.Element => {
             wrapperCol={{ span: 24 }}
             validateTrigger="onBlur"
             onFinish={onFinish}
+            onValuesChange={(_, values) => {
+              setBalanceBaseAmount(values['balanceBaseAmount']);
```

Comment:
Both setters will run if any input value change?
Can we move setters to `setInputValue` instead?

Reply:
Check which value is updated and call only corresponding setter